xonsh/environ.py

```diff
@@ -1689,7 +1689,12 @@ class PTKSetting(PromptSetting):  # sub-classing -> sub-group
     )
     XONSH_COPY_ON_DELETE = Var.with_default(
         False,
-        "Whether to copy words/lines to clipboard on deletion (must be set in .xonshrc file)."
+        "Whether to copy words/lines to clipboard on deletion (must be set in .xonshrc file). (deprecated)"
+        "Only available under the prompt-toolkit shell.",
```

Comment:
`.xonshrc` can be changed to something generic. The config file can live in multiple places.